server.js

```diff
@@ -30,6 +32,26 @@ const makeSgRequest = body =>
     body: body.toJSON()
   })
 
+// This converts {a:1, b:2} into 'a=1&b=2'
+const queryParams = obj =>
+  Object.keys(obj)
+    .map(key => [key, obj[key]]) // There is no Object.entries() in node 6
+    .map(([key, val]) => `${key}=${val}`)
+    .join('&')
+
+const wrikeMkFolder = name =>
+  fetch(process.env.WRIKE_URL, {
+    body: queryParams({
```

Comment:
I tried `encodeURIComponent('=')` and `encodeURIComponent('&')` in a REPL and they were being encoded. I guess they would be decoded on the other side anyway, but it should work how it is right now.